classes/i18n/LocaleConversion.php

```diff
@@ -24,18 +24,41 @@
 namespace PKP\i18n;
 
 use Illuminate\Support\Arr;
+use PKP\core\Core;
 use PKP\facades\Locale;
 
 class LocaleConversion
 {
+    /**
+     * Get ISO639-2b array
+     */
+    protected static function getISO6392b(): array
+    {
+        $iso6392BFile = Core::getBaseDir() . '/' . PKP_LIB_PATH . '/lib/vendor/sokil/php-isocodes-db-i18n/databases/iso_639-2.json';
```

Comment:
I copied the file localy, to lib/pkp/lib/iso_639-2.json, and am falling back to it, as you suggested.

Reply:
Yeah, I was thinking about a kind of "build step", I just think the file should be committed to avoid surprises, like you depend on the "package^1.0", then composer might pick the 1.1 depending on how the person installed (which could break things).

Reply:
Hmmm... I am not sure if this could help/if it is the right solution... :thinking: 
Actually, we only need the copy of that file if sokil at some point stops providing it -- in that case when we/users get their code with composer we will not have it. 
As far as I understand those solutions with composer: they all get the remote files first and then do some things. Is this so? If so, it is not so useful for us, because if it gets the sokil library that has the file, no need to copy it. And if it gets the sokil library that does not have the file, we cannot copy it anyway.
Thus, actually, I think, we would need a script, that would download only that file if there is no such file in the downloaded sokil library. Could we then implement it so in that PHP function?
Also, this file is actually the file from debian package. So maybe there is another way for us to keep and maintain it locally?
:thinking: